website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
 RISC Zero provides an already deployed verifier contract in the Sepolia network for your convenience.
 You can choose to use this contract or deploy your own.
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
+| ----------------------------- | ---------------------------- | --------- | ---------------------------------------------------------------------- |
+| [RiscZeroGroth16Verifier.sol] | [v0.9.0][contracts-v0.9.0]   | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier-v0.9.0]  |
```

Comment:
Currently clicking on either of these contract links will land you on the v0.8 version of the contract.

Reply:
As for the deployed contracts, I've realized that this is a "corporate needs you to find the difference between this contract and this contract" situation.... there were no changes to the verifier between 0.9 and 0.10. (Evidence `git diff v0.9.0..v0.10.0 --stat -- contracts`)